### DISCUSSION TITLE
Sorry, I didn't understand this version!

### DISCUSSION THREAD

**Discussion by ghost** [goharbor/harbor#15908] [Q&A] [answered]
Which is new, v1.10.9 or v2.4.0?

**Comment by juliohm1978** [comment]
They are both new.
* 2.4.0 is the latest.
* 1.10.9 is a security patch

**Reply by ghost** [reply]
Does v2.4.0 include the security patch of v1.10.9?

**Comment by wy65701436** [comment, marked as answer]
Harbor v2.4.0 is the latest release build.

Actually, the v1.10.9 is a patch branch release-1.10 for particular case, if you're a user that already use the Harbor v2.0+, you can just ignore it.

For the security concern, we do the security scan before any release, no worry.

We will arrange the release in future to avoid the confuse.

**Reply by wy65701436** [reply]
cc @xaleeks

**Reply by ghost** [reply]
thank a lot
